uiplib/gui/mainGui.py

```diff
@@ -94,6 +96,7 @@ def flush(self):
                                 icon='warning')
         if ask == 'yes':
             flush_wallpapers(self.settings['pics-folder'])
+            self.update_pics_from_fav_pics()
```

Comment:
I dont think this needs to be there. When we click flush, only images is affected. Fav pics wont be affected and you dont need to update it again after a flush.